news/2018-06-12-the-qat-gazette-may-2018.md

```diff
@@ -0,0 +1,97 @@
+
+I have both osu! related and personal questions prepared for the QAT members who elect to be interviewed, but we also want to give you, the community, the chance to discover! À la the “Ask Me Anything” feature present in the original QAT blogs, you can leave your questions into the Disqus comments below and I will pick some of them to use in the interviews.
+
+I hope you are excited for it as I am, because I have many more plans should the QAT interviews be a success!
+
+### QAT Leader Elections
+by [Asherz007](https://osu.ppy.sh/users/9014047)
+
+With the loss of Okoratu a couple of months back, JBHyperion was left to lead the QAT by himself for a while. With JBH soon to get his hands tied down with the 2018 osu!catch World Cup, he took it upon himself to hold a re-election for the position of QAT Leader, in which any member of the QAT was able to run, similar to the election following the Upheaval discussion almost a year ago.
+
+Just short of three weeks ago, the [election results](https://osu.ppy.sh/community/forums/topics/749819) were released, and a new QAT leader was born! Alongside JBH, offer your congratulations to [Mao](https://osu.ppy.sh/users/2204515), who will join in captaining the team and push us forward into new and uncharted territories!
+
+### New Quality Assurance Team Additions
+This month saw one addition amongst our ranks, so say hello to [Chaoslitz](https://osu.ppy.sh/users/3621552), who joins the disqualification branch for the osu! game mode in order to help keep those circles and sliders in tip top condition! Chaoslitz will also participate in the evaluation of osu! Beatmap Nominator applications when the next cycle begins towards the end of this month.
+
+### Quality Assurance Team Retirements
+As with anything in life, circumstances change and people must move on and adapt. This month we bid a second farewell to [Myxomatosis](https://osu.ppy.sh/users/2202645), who has helped us tremendously in maintaining the quality of osu! maps and evaluating Beatmap Nominator applications since his return in January. We once again wish you the best in your future endeavours!
+
+## Summary
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+So once again we reach the end of another month’s update, knowing that tomorrow brings both new challenges and new opportunities. The outlook of next month’s article may vary depending on my commitments to the 2018 osu!catch World Cup, so don’t be surprised if you see someone else leading the charge in June’s edition!
+
+One thing you can bet your hat on though, is that it’ll be chock full of all the latest and greatest news just as always, so please look forward to it! Until next time!
```

Comment:
google leads me to believe that "chock full" needs a hyphen between the words. not sure though, I've seen it written both ways

Reply:
I've personally never seen it used this way but I guess I've been doing it wrong lol